tee-worker/litentry/primitives/src/lib.rs

```diff
@@ -49,10 +48,11 @@ pub mod sgx_reexport_prelude {
 
 use rand::Rng;
 
-pub use assertion::*;
 pub use enclave_quote::*;
 pub use validation_data::*;
 
+pub use parentchain_primitives::*;
```

Comment:
Let's try to avoid importing `*` but only those we actually need. Plus it shadows line 28. I'm just concerned that we expose more than what we need, and use the wrong primitives.